packages/access-policy/package.json

```diff
@@ -13,11 +13,14 @@
   "types": "dist/index.d.ts",
   "scripts": {
     "build": "rm -rf dist && run-p \"build:* {@}\" --",
-    "build:src": "babel src --out-dir dist --source-maps --config-file \"../../babel.config.json\"",
-    "build:types": "tsc",
-    "lint": "eslint --ignore-path ../../.gitignore .",
+    "build:src": "yarn package:build:js",
+    "build:types": "yarn package:build:types",
+    "lint": "yarn package:lint:js",
     "lint:fix": "yarn lint --fix",
-    "test": "jest",
+    "test": "yarn package:test",
     "test:coverage": "yarn test --coverage"
+  },
+  "devDependencies": {
+    "npm-run-all": "^4.1.5"
```

Comment:
With Yarn2+, each package must explicitly declare its dependencies. 
See https://yarnpkg.com/features/workspaces/#what-does-it-mean-to-be-a-workspace